Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -229,24 +234,27 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    continueonerrortype: ""
+    isoversize: false
+    isautoswitchedtoquietmode: false
   "6":
     id: "6"
-    taskid: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+    taskid: d967878f-480e-4e27-8eb0-a193826cd386
     type: regular
     task:
-      id: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+      id: d967878f-480e-4e27-8eb0-a193826cd386
       version: -1
-      name: Get CVE info from "CVE Search"
+      name: Get CVE info from CIRCL CVE Search
       description: Returns CVE information by CVE ID.
-      script: '|||cve'
+      script: 'CIRCL CVE Search|||cve'
       type: regular
       iscommand: true
-      brand: CVE Search
+      brand: CIRCL CVE Search
```

Comment:
CVE search and CVE are generic command.
The command will fail if there are no integrations that support the command. 

Old integrations use the "!cve_search", new integration use the "!cve" command.
Lets remove the integration check + the brand specific with the "skip if unavailable"

Reply:
fixed - removed all integration check tasks, the commands were replaced with the generic CVE search and CVE.